tests/model_serving/model_server/maas_billing/conftest.py

```diff
@@ -60,19 +71,35 @@ def minted_token(request_session_http, base_url: str, current_client_token: str)
     return token
 
 
-@pytest.fixture(scope="module")
-def base_url(admin_client) -> str:
-    scheme = detect_scheme_via_llmisvc(client=admin_client)
+@pytest.fixture(scope="class")
+def base_url(
+    admin_client: DynamicClient,
+    unprivileged_model_namespace: Namespace,
+) -> str:
+    ns = unprivileged_model_namespace.name
+    scheme = detect_scheme_via_llmisvc(
+        client=admin_client,
+        namespace=ns,
+    )
     host = host_from_ingress_domain(client=admin_client)
     return f"{scheme}://{host}/maas-api"
 
 
-@pytest.fixture(scope="session")
-def model_url(admin_client) -> str:
+@pytest.fixture(scope="class")
+def model_url(
+    admin_client: DynamicClient,
+    unprivileged_model_namespace: Namespace,
+    maas_inference_service_tinyllama: LLMInferenceService,
+) -> str:
     """
     MODEL_URL:http(s)://<host>/llm/<deployment>/v1/chat/completions
+
     """
-    scheme = detect_scheme_via_llmisvc(client=admin_client)
+    ns = unprivileged_model_namespace.name
+    scheme = detect_scheme_via_llmisvc(
+        client=admin_client,
+        namespace=ns,
+    )
```

Comment:
May be create a fixture? I see you are using the same exact call in two places, values won't be different for these.

Reply:
Can you please remove the line saving ns?